### PR TITLE
RD-4651: Simplified condition for displaying the ShowFirstUserJourneyButtons component

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Simplified condition used for displaying the `ShowFirstUserJourneyButtons` component.
Now the component should be displayed if any deployment is being returned from the backend.

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/2593/)

## Documentation

[PR](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2182) for the documentation update has been created.
The sychronization should take place as a part of this PR.

### CODE REVIEW

**Comment by NorbertAmartus** [comment]
Because of the @kubama sick leave, I'm adding another reviewer so that we could proceed with the PR

File: widgets/deployments/src/widget.tsx [hunk 1]
```diff
@@ -166,12 +149,16 @@ Stage.defineWidget({
 
     render(widget, data, error, toolbox) {
         const { Loading } = Stage.Basic;
+        const {
+            configuration: { showFirstUserJourneyButtons }
+        } = widget;
+        const shouldShowFirstUserJourneyButtons = showFirstUserJourneyButtons && _.isEmpty(data?.items);
```
Comment (qooban) [inline widgets/deployments/src/widget.tsx:155]: Optional:

Maybe use native `length` property? Or convert `lodash` uses in this file to explicit imports (Ref. [Ongoing Refactors in Cloudify UI - Using explicit imports in widgets](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/2148073473/Ongoing+Refactors+in+Cloudify+UI#Using-explicit-imports-in-widgets))?

**Review by qooban — APPROVED** [review]

File: widgets/deployments/src/widget.tsx (hunk 1 above)
Comment (NorbertAmartus) [reply widgets/deployments/src/widget.tsx]: Done accordingly to our slack conversation 🎉

**Review by qooban — APPROVED** [review]
LGTM 👍🏻 

Thanks for additional effort to address optional comments 🙂

**Pull request merged**